github.md

```diff
@@ -22,3 +32,69 @@ you should do:
+In most (all) cases, pull requests should be reviewed by at least one other developer. 
+
+When creating a new pull request, you need to pick what branch you're merging - usually your dev branch - and what you're aiming to merge into - usually master, create a logical title, and add a description that includes: 
+* required: an explanation of what the reason/function of the code in the PR (the "why")
+* required: an explanation of what code changed in the PR (the "what")
+* optional: Link(s) to view changes, associated JIRA tickets, or other links that may be useful for understanding the changes in the PR.
+* optional: mention any person who may need to be aware of the changes in the PR (PM, UX, BO, or other devs)
+
+You should then assign one or more reviewers. (If you have more than one, you may want to mention in the description what each should be reviewing).
+
+Other notes: 
+"WIP" indicates a PR that is a work-in-progress that is not ready for review, but can be useful for comparing diffs as you work, or sharing in-progress work with others to see or comment on. In general, WIP PRs can be ignored.
+
+## Github and JIRA
+
+In general, if you are creating code on GitHub, commenting and review should happen there, with the code. If you are also using JIRA, you should add a comment with a link to the PR to the story in JIRA and move the story to the Review column to indicate that the PR is under review.
+
+(If changes are not commited to GitHub - like with Wordpress configurations - the comments and review should happen in JIRA; Mention reviewers by name in a comment on the ticket, what they should be reviewing, where they can view the changes, and then move the story to the Review column.)
+
+- - -
+
+## GitHub basics
+
+This process assumes you have: a terminal program (like iTerm with oh-my-zsh), an editor program (like Sublime Text or Atom), a github account and access to the MIT Libraries repos, and have already cloned the repos to your local machine. (For help with those steps, contact one of the MITLib devs.)
+
+*Start fresh*: Checkout master branch
+```git checkout master```
+
+*Pull*: Get the latest changes that have been merged 
+```git pull```
+
+*Create a branch*: Create a new branch starting with master (or the branch you want to base your work on): This is the clean separate space where you will put your work.
+```git checkout -b [branchname]```
+
+*Do stuff*: open the repo in your editor and write code or make edits that relate to the fix/feature/experiment you are working on.
+
+*Commit your changes regularly*: once you have a few changes that add up to something (eg. 'added new js function to load rss' or 'added new css to support adjusted mini-cal alignment') or if you have to stop for a bit or switch to something else, add the files you are ready to commit, then commit them to your branch and push them to github.
+
+```
+git add [filename] 
+git add [filename] 
+...
+git commit -m 'short description of what is in the commit'
```

Comment:
I would recommend not using the `-m` switch, unless your commit description is really just a one-liner. A good commit message should have a short description on the first line, a blank line, and then a longer description. With the `-m` switch you only get the first line.